pact/coin-contract/v2/coin.pact

```diff
@@ -326,6 +337,21 @@
       (credit account account-guard amount))
     )
 
+  (defun remediate:string (account:string amount:decimal)
+    @doc "Slash allows the termination of coins. This function \
```

Comment:
```suggestion
    @doc "Remediate an incorrect coinbase or other credit. This function \
```

Reply:
Odd - I think this was merged incorrectly and overwrote my doc changes